### ISSUE TITLE
测试issue

### ISSUE THREAD

**Issue by pengfei** [pengfei/blog#1]
### 测试issue
#TO DO 
怎么在issue中插入图片？
![bfc243a3gy1g4dkwfev15j215o0rsu0x](https://user-images.githubusercontent.com/54535691/63836501-4161d800-c9ac-11e9-99d2-0ccd75c2ccd8.jpg)
`import numpy as np
np.random.seed(1337)  # for reproducibility
from keras.models import Sequential
from keras.layers import Dense
import matplotlib.pyplot as plt # 可视化模块
`
[1234.hi.cn](url)

插入代码
`import numpy as np
np.random.seed(1337)  # for reproducibility
from keras.models import Sequential
from keras.layers import Dense
import matplotlib.pyplot as plt # 可视化模块`

插入的代码怎么换行？？ #

**Comment by pengfei** [comment]
<pre>np.random.seed(1337)  # for reproducibility
from keras.models import Sequential
from keras.layers import Dense
import matplotlib.pyplot as plt # 可视化模块 </pre>

**Comment by pengfei** [comment]
```
np.random.seed(1337)  # for reproducibility
from keras.models import Sequential
from keras.layers import Dense
import matplotlib.pyplot as plt # 可视化模块 

```

**Comment by pengfei** [comment]
```python
np.random.seed(1337)  # for reproducibility
from keras.models import Sequential
from keras.layers import Dense
import matplotlib.pyplot as plt # 可视化模块
```

**Comment by pengfei** [comment]
```c++
#include  <stdio.h>
    int main(void)
    {
        printf("Hello world\n");
    }
```

**Comment by pengfei** [comment]
> ### 测试issue
> #TO DO
> 怎么在issue中插入图片？
> ![bfc243a3gy1g4dkwfev15j215o0rsu0x](https://user-images.githubusercontent.com/54535691/63836501-4161d800-c9ac-11e9-99d2-0ccd75c2ccd8.jpg)
> `import numpy as np np.random.seed(1337) # for reproducibility from keras.models import Sequential from keras.layers import Dense import matplotlib.pyplot as plt # 可视化模块 `
> [1234.hi.cn](url)
> 
> 插入代码
> `import numpy as np np.random.seed(1337) # for reproducibility from keras.models import Sequential from keras.layers import Dense import matplotlib.pyplot as plt # 可视化模块`
> 
> 插入的代码怎么换行？？

**Comment by pengfei** [comment]
![image](https://user-images.githubusercontent.com/54535691/63837250-b255bf80-c9ad-11e9-9327-0a2944bade44.png)

**Comment by pengfei** [comment]
https://mp.weixin.qq.com/s/V4sK4L6j0Dl2QsJFJTIf7w
